modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -27,6 +27,7 @@ String analyticsReportsPanelState = SessionClicks.get(request, "com.liferay.anal
 		<span class="font-weight-bold"><liferay-ui:message key="content-performance" /></span>
 
 		<clay:button
+			aria-label="Close"
```

Comment:
Same here, I think you can use LanguageUtil as [here ](https://github.com/brianchandotcom/liferay-portal/blob/master/modules/apps/commerce/commerce-address-web/src/main/resources/META-INF/resources/view.jsp#L65). Don't forget to check if close language key is defined :)